Source/WebCore/rendering/RenderLayer.h

```diff
@@ -739,11 +739,12 @@ class RenderLayer : public CanMakeWeakPtr<RenderLayer> {
     TransformationMatrix currentTransform(OptionSet<RenderStyle::TransformOperationOption> = RenderStyle::allTransformOperations) const;
     TransformationMatrix renderableTransform(OptionSet<PaintBehavior>) const;
     
-    // Get the perspective transform, which is applied to transformed sublayers.
-    // Returns true if the layer has a -webkit-perspective.
+    // Get the children transform (to apply a perspective on children), which is applied to transformed sublayers, but not this layer.
+    // Returns true if the layer has a perspective.
     // Note that this transform has the perspective-origin baked in.
-    TransformationMatrix perspectiveTransform(const LayoutRect& layerRect) const;
+    TransformationMatrix childrenTransform() const;
```

Comment:
Sadly I don't like the new naming. The only use for this transform is for perspective, and the fact that it's a child transform is a detail of the compositing architecture. Here in rendering code it should be called `perspectiveTransform`. My previous question was more along the lines of "should this be called `perspectiveTransformIncludingTransofrmOrigin`".

Reply:
Ah, ok, I will adapt this soon, misunderstood you.